### PR TITLE
Hey guys a few changes as per ticket #171694768

### PR DESCRIPTION
**Pull request by Roakz**
- Placeholder dates are as they originally were as discuseed
- Changed the side navigation bar so that items are above the series as requested by QSA.
- Removed the description from the series search result card.

### CODE REVIEW

File: qsa-public-spa/src/recordViews/Results.tsx [hunk 1]
```diff
@@ -414,7 +414,12 @@ const SearchResult: React.FC<{ searchResult: any }> = props => {
           {props.searchResult.qsa_id_prefixed}
         </span>
       </div>
-      {props.searchResult.description && <p dangerouslySetInnerHTML={{__html: preserveNewLines(props.searchResult.description)}} />}
+      {console.log(props.searchResult.json)
```
Comment (jambun) [inline qsa-public-spa/src/recordViews/Results.tsx]: drop this debug line

File: qsa-public-spa/src/advancedSearch/AdvancedSearch.tsx [hunk 2]
```diff
@@ -162,7 +162,7 @@ const AspaceAdvancedSearch: React.FC<{
                       className="form-control"
                       type="text"
                       name="from"
-                      placeholder="yyyy, yyyy-mm or yyyy-mm-dd"
+                      placeholder="yyyy, mm-yyyy or dd-mm-yyyy"
```
Comment (jambun) [inline qsa-public-spa/src/advancedSearch/AdvancedSearch.tsx]: The rest of the app is expecting yyyy-mm-dd so this won't do the trick. Maybe best to drop this commit and we can deal with it separately.

Comment (Roakz) [reply qsa-public-spa/src/advancedSearch/AdvancedSearch.tsx]: Thanks for the feedback. I have removed the console.log and I'm just waiting to hear back from Megan about what they want to do about the date before I fix it up.

Comment (Roakz) [reply qsa-public-spa/src/advancedSearch/AdvancedSearch.tsx]: Hey James...Im guessing the date issue is that it doesn't accept the date range the other way around? as in the placeholder formatt is the only way to get accurate results?

Comment (Roakz) [reply qsa-public-spa/src/advancedSearch/AdvancedSearch.tsx]: Hey, James, I have made a new commit with a reference to the original and reverted the date placeholders back... I can see that the search accuracy relies on that format. Megan will probly talk to you guys in the future depending on what QSA decides on the dates. The other 2 fixes are still valid and required. cheers.

File: qsa-public-spa/src/recordViews/Results.tsx (hunk 1 above)
Comment (Roakz) [reply qsa-public-spa/src/recordViews/Results.tsx]: removed

**Comment by jambun** [comment]
Thanks @Roakz

**Pull request merged**